en/flight_stack/controller_diagrams.md

```diff
@@ -63,11 +63,13 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
 
 #### Combined Position and Velocity Controller Diagram
 
-![MC Position Controller Diagram](../../assets/diagrams/px4_mc_position_controller_diagram.png)
+![MC Position Controller Diagram](../../assets/diagrams/mc_combined_position_velocity_controller_diagram.png)
```

Comment:
Ah, name reverted but not in the docs.

Reply:
Oh my bad sry

Reply:
No worries! Thanks for fixing up the diagram!